src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/dto/TierDto.kt

```diff
@@ -4,36 +4,37 @@ import com.fasterxml.jackson.annotation.JsonCreator
 import com.fasterxml.jackson.annotation.JsonProperty
 import io.swagger.annotations.ApiModel
 import io.swagger.annotations.ApiModelProperty
-import uk.gov.justice.digital.hmpps.hmppstier.domain.enums.ChangeScore
-import uk.gov.justice.digital.hmpps.hmppstier.domain.enums.ProtectScore
+import uk.gov.justice.digital.hmpps.hmppstier.domain.enums.ChangeLevel
+import uk.gov.justice.digital.hmpps.hmppstier.domain.enums.ProtectLevel
 import uk.gov.justice.digital.hmpps.hmppstier.jpa.entity.TierCalculationResultEntity
 
 @ApiModel(description = "Tier")
 data class TierDto @JsonCreator constructor(
 
-  @ApiModelProperty(value = "Protect Tier", example = "D")
-  @JsonProperty("protectTier")
-  val protectTier: ProtectScore,
+  @ApiModelProperty(value = "Protect Level", example = "D")
+  @JsonProperty("protectLevel")
+  val protectLevel: ProtectLevel,
 
-  @ApiModelProperty(value = "Protect Score", example = "17")
-  @JsonProperty("protectScore")
-  val protectScore: Int?,
+  @ApiModelProperty(value = "Protect Points", example = "17")
+  @JsonProperty("protectPoints")
+  val protectPoints: Int?,
 
-  @ApiModelProperty(value = "Change Tier", example = "2")
-  @JsonProperty("changeTier")
-  val changeTier: ChangeScore,
+  @ApiModelProperty(value = "Change Level", example = "2")
```

Comment:
The example looks like it should be "TWO" rather than "2" according to the ChangeLevel enum